mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule Stripy.Mixfile do
   end
 
   defp deps do
-    [{:httpoison, "~> 0.13"},
+    [{:httpoison, "~> 1.0"},
```

Comment:
Let's make this `~> 0.13 or ~> 1.0` for users still on HTTPoison 0.13.

Reply:
@toddholmberg Yes. https://github.com/appsignal/appsignal-elixir/blob/develop/mix.exs#L92